pkg/storj/node.go

```diff
@@ -3,13 +3,211 @@
 
 package storj
 
-import "encoding/hex"
+import (
+	"crypto"
+	"crypto/x509"
+	"encoding/base64"
+	"fmt"
+	"math/bits"
+
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+	"golang.org/x/crypto/sha3"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/utils"
+)
+
+// IdentityLength is the number of bytes required to represent a node id
+const IdentityLength = 32
+
+// ErrNodeID is used when something goes wrong with a node id
+var (
+	ErrNodeID   = errs.Class("node ID error")
+	EmptyNodeID = nodeID([IdentityLength]byte{})
+)
+
+type Node struct {
```

Comment:
Let's just change only NodeID, the Node change needs to be done such it doesn't use protobuf. protobuf is not valuable in code design. The goal is to push protobuf to the edges such that it preserves clarity on what is the implementation.